tests/foreman/api/test_subscription.py

```diff
@@ -232,7 +232,7 @@ def test_positive_delete_manifest_as_another_user(function_org):
 
 
 @pytest.mark.tier2
-def test_positive_subscription_status_disabled(module_ak, rhel7_contenthost, module_org):
+def test_positive_subscription_status_disabled(module_ak, rhel_contenthost, module_org):
```

Comment:
@ColeHiggins2 do you think this is a good test to parametrize rhel versions for this test case?